src/atomic_ops/sysdeps/gcc/sw_64.h

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
I see that you've copied and edited gcc/alpha.h file. Is sw_64 very close to Alpha architecture regarding memory consistency? Why not just include "generic.h" and that's all? (see e.g. gcc/e2k.h)

Reply:
sw_64 and alpha architecture are not the same. Therefore it needs to be modified like this